packages/shared/locales/en.json

```diff
@@ -191,7 +191,7 @@
         },
         "ledgerInstallationGuide": {
             "title": "Have you installed the necessary Ledger apps?",
-            "body1": "Before you proceed, you must find and install both the {legacy} app and the new IOTA app through Ledger Live.",
+            "body1": "Before you proceed, you must use Ledger Live to find and install {protocol} app.",
```

Comment:
Can we add a "the" before protocol, e.g. `...to find and install the {protocol} app`?